lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -73,6 +73,14 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		return await this.driver.findElement( textSelector ).sendKeys( text );
 	}
 
+	async insertContactForm() {
+		await this.addBlock( 'Shortcode' );
+		const shortcodeSelector = By.css( 'textarea.editor-plain-text' );
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, shortcodeSelector );
+		const shortcodeTextarea = this.driver.findElement( shortcodeSelector );
+		shortcodeTextarea.sendKeys( '[contact-form][/contact-form]' );
```

Comment:
@JavonDavis Sorry for the late review! 

We should await `findElement` and `sendKeys` since they are returning premises. And return `sendKeys` also.

Reply:
🤦‍♂️ You're right I'll send through a patch